templates/download/server/linuxone.html

```diff
@@ -23,10 +23,10 @@
             <div class="twelve-col no-margin-bottom">
```

Comment:
Page tags should be: "IBM, LinuxONE, Ubuntu Server, OpenStack, Cloud, Ubuntu OpenStack, z Systems, Mainframe"